canonical/specs-v1.canonical.com
# Add update command

## Done

- Add a `update-spreadheet` command that... updates the spreadsheet (same behaviour than the google script)
- Add configs to deploy a cronjob
- Point to another spreadsheet: [Specs tracker](https://docs.google.com/spreadsheets/d/1aKH6petyrzjzw0mgUNQscDhFSfVkbAIEjfH7YBS-bDA/edit#gid=0) (that will be the "official" if this is merged)

## QA
- `dotrun exec flask update-spreadsheet`
- Wait till the script finishes
- Check that the spreadsheet is filled
- `dotrun`
- Check that the website is not broken

## Issue
Fixes https://github.com/canonical/workplace-engineering-squad/issues/864
